packages/framework/src/Component/ConfirmDelete/ConfirmDeleteResponseFactory.php

```diff
@@ -27,7 +27,7 @@ public function __construct(
      * @param mixed $entityId
      * @return \Symfony\Component\HttpFoundation\Response
      */
-    public function createDeleteResponse($message, $route, $entityId)
+    public function createDeleteResponse(string $message, string $route, mixed $entityId): Response
```

Comment:
Practically, does it make sense to pass anything other than `int` or `string`as `$entityId`?